src/utils/ota/ota.go

```diff
@@ -2,80 +2,41 @@
 package ota
 
 import (
-	"bufio"
-	"fmt"
-	"log"
-	"os"
-
 	"github.com/blang/semver"
+	"github.com/pkg/errors"
```

Comment:
This package is deprecated.

Reply:
That's interesting, didn't know about it.